client/hosting/sites/components/sites-dataviews/style.scss

```diff
@@ -40,8 +41,8 @@
 	white-space: nowrap;
 }
 
-.sites-dataviews__site-url,
```

Comment:
Scoping these styles to the table layout, has the following side-effect (notice how links in the list layout end up with a bluish color):

Before

https://github.com/user-attachments/assets/882230e0-c558-4687-9484-889a58d6897e

After

https://github.com/user-attachments/assets/54ae47bd-2571-4fb0-9e95-6d01f88d31d7

Reply:
Pushed a style fix for this at https://github.com/Automattic/wp-calypso/pull/93503/commits/5dee9872d985ed360719cea43b64229fcd7a6ba5 so the link color stays stable across layouts:

https://github.com/user-attachments/assets/894e8e81-7404-45bd-9206-b91dd1363073